ValidateProfileEmail/handler.ts

```diff
@@ -160,6 +167,25 @@ export const ValidateProfileEmailHandler = (
     );
   }
 
+  // Check if the e-mail is already taken
+  if (FF_UNIQUE_EMAIL_ENFORCEMENT_ENABLED(fiscalCode)) {
```

Comment:
We should move this logic after the profile and email check, just before the profile update.
This because the validation email link may refers to an old profile or a previous validation, and the email uniqueness error will be outdated on the current data.

Reply:
Fixed in [a328180](https://github.com/pagopa/io-functions-public/pull/67/commits/a3281809929880eb4e0bb08479cc16c14348c76f)